tests/base/parameterframe/test_parameterframe_annotations.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from bluemira.base.parameter_frame import Parameter, ParameterFrame
+
+
+@dataclass
+class PFrame(ParameterFrame):
+
+    a: Parameter[float]
+    b: Parameter[int]
+
+
+def test_future_annotations_are_typed():
```

Comment:
I think a comment on the context/purpose of this test would be good. I don't think the purpose is very clear from just this code. 

Really, the purpose of the test is to check we can still do type validation when we've imported `annotations`. This kind of reads like we're testing that `annotations` converts types to strings.